### DISCUSSION TITLE
2010/12/06/%E8%B1%86%E8%B1%86%E5%87%BA%E7%94%9F%E8%AE%B0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#304] [Announcements]
# 2010/12/06/%E8%B1%86%E8%B1%86%E5%87%BA%E7%94%9F%E8%AE%B0

当爸爸的心情是非常激动的。

https://blog.qizhen.xyz/2010/12/06/%E8%B1%86%E8%B1%86%E5%87%BA%E7%94%9F%E8%AE%B0

**Comment by ruanqizhen** [comment]
之前博客下的留言：

foolooo
2010年12月14日 08:17 编辑
恭喜阮工，喜得贵豆:)

回复
